stm32-data-gen/src/interrupts.rs

```diff
@@ -477,6 +477,7 @@ fn valid_signals(peri: &str) -> Vec<String> {
         ("I3C", &["ER", "EV", "WKUP"]),
         ("FMPI2C", &["ER", "EV"]),
         ("TIM", &["BRK", "UP", "TRG", "COM", "CC"]),
+        ("LPTIM", &["BRK", "UP", "TRG", "COM", "CC"]),
```

Comment:
There's no chip that splits LPTIM interrupts into several interrupt lines for the separate events (break, update, trigger, etc), so I don't think we should add this.

If you don't add it then LPTIMs get a single interrupt signal "GLOBAL", which you can still use from embassy-stm32.

Reply:
Agreed. I wasn't quite sure about this to begin with, which is why I put it in its own commit. Just nuked it 😄